components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -62,15 +71,22 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
   }, [isFormOpenMobile]);
 
   const handleGenerateHighlightSummary = async () => {
-    if (!pullrequestLink || !isValidPullRequestUrl(pullrequestLink)) {
-      toast({ description: "Please provide a valid pull request link!", title: "Error", variant: "danger" });
+    if (!pullrequestLink || (!isValidPullRequestUrl(pullrequestLink) && !isValidIssueUrl(pullrequestLink))) {
+      toast({ description: "Please provide a valid issue or pull request link!", title: "Error", variant: "danger" });
       return;
     }
 
     setIsSummaryButtonDisabled(true);
 
-    const commitMessages = await getPullRequestCommitMessageFromUrl(pullrequestLink);
-    const summary = await generatePrHighlightSummaryByCommitMsg(commitMessages);
+    var summary: string | null;
```

Comment:
Usage of `var` is not allowed. Always use `let` or `const`

```suggestion
    let summary: string | null;
```